generator.rb

```diff
@@ -0,0 +1,57 @@
+def is_vowel?(letter)
+  if letter == "a" || letter == "e" || letter == "u" || letter == "o" || letter == "i"
+    return true
+  else
+    return false
+  end
+end
+
+def run_generator
+  puts "Please enter the first word:"
+  first_word = gets.chomp
+  until first_word.length >= 2
+    puts "please enter a word with more than one character:"
+    first_word = gets.chomp
+  end
+  puts "please enter the second word:"
+  second_word = gets.chomp
+  until second_word.length >= 2
+    puts "please enter a word with more than one charactor:"
```

Comment:
You have almost exactly the same code to get an input word from the user repeated here. Could you DRY this up with another method?